app/views/placements/providers/placements/index.html.erb

```diff
@@ -0,0 +1,120 @@
+                    <%= t(".phase") %>
+                  </dt>
+                  <dd class="app-result-detail__value">
+                    <%= placement.school.phase %>
+                  </dd>
+                </div>
+                <div class="app-result-detail__row">
+                  <dt class="app-result-detail__key">
+                    <%= t(".age_range") %>
+                  </dt>
+                  <dd class="app-result-detail__value">
+                    <%= placement.age_range %>
+                  </dd>
+                </div>
+                <div class="app-result-detail__row">
+                  <dt class="app-result-detail__key">
+                    <%= t(".school_type") %>
+                  </dt>
+                  <dd class="app-result-detail__value">
+                    <%= placement.school.type_of_establishment %>
+                  </dd>
+                </div>
+                <div class="app-result-detail__row">
+                  <dt class="app-result-detail__key">
+                    <%= t(".gender") %>
+                  </dt>
+                  <dd class="app-result-detail__value">
+                    <%= placement.school.gender %>
+                  </dd>
+                </div>
+                <div class="app-result-detail__row">
+                  <dt class="app-result-detail__key">
+                    <%= t(".religious_character") %>
+                  </dt>
+                  <dd class="app-result-detail__value">
+                    <%= placement.school.religious_character %>
+                  </dd>
+                </div>
+                <div class="app-result-detail__row">
+                  <dt class="app-result-detail__key">
+                    <%= t(".urban_or_rural") %>
+                  </dt>
+                  <dd class="app-result-detail__value">
+                    <%= placement.school.urban_or_rural %>
+                  </dd>
+                </div>
+                <div class="app-result-detail__row">
+                  <dt class="app-result-detail__key">
+                    <%= t(".ofsted_rating") %>
+                  </dt>
+                  <dd class="app-result-detail__value">
+                    <%= placement.school.rating %>
+                  </dd>
+                </div>
+                <div class="app-result-detail__row">
+                  <dt class="app-result-detail__key">
+                    <%= t(".mentor") %>
+                  </dt>
+                  <dd class="app-result-detail__value">
+                    <%= placement.mentor_names %>
+                  </dd>
+                </div>
+                <div class="app-result-detail__row">
+                  <dt class="app-result-detail__key">
+                    <%= t(".trainee") %>
+                  </dt>
+                  <dd class="app-result-detail__value">
+                    <%= placement.trainee %>
+                  </dd>
+                </div>
+              </dl>
+            </li>
```

Comment:
This would be a good candidate for pulling out into a ViewComponent.

Then we could [render the collection](https://viewcomponent.org/guide/collections.html) – for example, something like:

```ruby
<%= render(Placements::PlacementSummary.with_collection(@placements)) %>
```